python/tests/test_collections.py

```diff
@@ -26,7 +31,11 @@
 COLLECTION_REFERENCE_ID = "test_id"
 NESTED_COLLECTION_REFERENCE_KEY = "nested_test_key"
 NESTED_COLLECTION_REFERENCE_ID = "nested_test_id"
-COLECTION_DOCUMNET_KEY = "test_document"
+COLLECTION_DOCUMNET_KEY = "test_document"
+DOCUMENT_DATA = {"test": "test"}
+BASE64_DOCUMENT_DATA = dict_to_base64(DOCUMENT_DATA)
+DOCUMENT_ID = "915b75c5-9e95-4fa7-aaa2-2214c8d251ce"
+KWARGS = {"headers": {"Authorization": "Bearer token"}}
```

Comment:
Rename this `HEADERS_WITH_AUTHORIZATION`, so that it is clear in the test code why it is used :)